CHANGELOG.md

```diff
@@ -9,9 +9,21 @@ Instructions: Add a subsection under `[Unreleased]` for additions, fixes, change
 
 ## [Unreleased]
 
+### Added
+
+- `pretext update` command will update managed files and the `requirements.txt` file to match current installed version of PreTeXt.
```

Comment:
Compare with `pretext init -r`

Reply:
`pretext init -r` does do the same as `pretext update -f`.  There isn't any extra code to maintain, but I'll put something in help to point out that `--refresh` will eventually be depricated.